Assets/Project/Scripts/Modules/GamePlayScene/GamePlayDirector.cs

```diff
@@ -34,13 +34,26 @@ public class GamePlayDirector : SingletonObject<GamePlayDirector>
 
         private const string _STAGE_NUMBER_TEXT_NAME = "StageNumberText";
         private const string _TIMER_TEXT_NAME = "TimerText";
-        private const string _PAUSE_WINDOW_NAME = "PauseWindow";
-        private const string _PAUSE_BACKGROUND_NAME = "PauseBackground";
         private const string _SUCCESS_POPUP_NAME = "SuccessPopup";
         private const string _FAILURE_POPUP_NAME = "FailurePopup";
 
         [SerializeField] private GameObject _tutorialWindow;
 
+        /// <summary>
+        /// 一時停止中の背景
+        /// </summary>
+        public GameObject pauseBackground;
```

Comment:
```suggestion
       [SerializeField] private GameObject pauseBackground;
```
ここは `private` にしましょう

Reply:
200e2ef